tests/golden_test_common.sh

```diff
@@ -137,20 +161,24 @@ popd > /dev/null
 
 if $has_errors; then
   cat >&2 << EOF
-Test failed.
+$(echo -e "${RED}")Golden test failed. Num passed: ${num_passed}/${num_files}$(echo -e "${NO_COLOR}")
+Files that caused diffs: ${files_with_diffs}
+Files with missing golden: ${files_missing_golden}
 
 # IF RUNNING ON GITHUB
-  Check the workflow step called "How to fix failing tests" for instructions on how to debug/fix this test.
+  Check below at the workflow step below called "How to fix failing tests" for instructions on how to debug/fix this test.
 
 # IF RUNNING LOCALLY
-  Look at the .xml.diff files in $OUTPUT_DIR
-
-  If the diffs are expected, update the golden files with:
-  cp "$OUTPUT_DIR"/*.xml "$GOLDEN_DIR"
+  View diffs with: (set DIFFTOOL to your favorite tool if needed)
+$(echo -e "${BLUE}")$diff_command_with_full_paths$(echo -e "${NO_COLOR}")
```

Comment:
Missing indent?

Reply:
The diff_command_with_full_paths variable actually contains the indents, because it's multiline so it's simpler this way. Added a comment above diff_command_with_full_paths, I can't really add a comment inside the heredoc.